### PR TITLE
Files for Openshift deployment templates and Github action changes

### PR DESCRIPTION
**Pull request by ssuganth**
# Description

This PR includes the following proposed change(s):

- Files for Openshift deployment templates and Github action changes

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Tested using my local repo.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by KFloodBCGov — APPROVED** [review]
Looks good, but we can wait for Suraiya or Ethan to review as well.

**Review by ekwanNTT — CHANGES_REQUESTED** [review]
Can you add a version annotation to the pod please which we can pull from the pipeline

File: gitops/openshift/rbac/template/imagepuller.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+OC_NAMESPACE=17b24e
```
Comment (KFloodBCGov) [inline gitops/openshift/rbac/template/imagepuller.env:1]: I've created a secret for this: OPENSHIFT_SA_NAMESPACE - should we use this instead?

File: gitops/openshift/apps/nginx/template/nginx_jag_scss.env [hunk 2]
```diff
@@ -0,0 +1,7 @@
+APP_NAME=jag-scss
+COMPONENT_NAME=nginx-jag-scss
+OC_NAMESPACE=17b24e
```
Comment (KFloodBCGov) [inline gitops/openshift/apps/nginx/template/nginx_jag_scss.env:3]: I've created a secret for this: OPENSHIFT_SA_NAMESPACE - should we use this instead?

File: gitops/openshift/apps/nginx/template/nginx_jag_scss_oldwm.env [hunk 3]
```diff
@@ -0,0 +1,6 @@
+APP_NAME=jag-scss
+COMPONENT_NAME=nginx-jag-scss-oldwm
+OC_NAMESPACE=17b24e
```
Comment (KFloodBCGov) [inline gitops/openshift/apps/nginx/template/nginx_jag_scss_oldwm.env:3]: I've created a secret for this: OPENSHIFT_SA_NAMESPACE - should we use this instead?

File: gitops/openshift/apps/scss-jag/template/scss_jag.env [hunk 4]
```diff
@@ -0,0 +1,7 @@
+APP_NAME=jag-scss
+COMPONENT_NAME=jag-scss
+OC_NAMESPACE=17b24e
```
Comment (KFloodBCGov) [inline gitops/openshift/apps/scss-jag/template/scss_jag.env:3]: I've created a secret for this: OPENSHIFT_SA_NAMESPACE - should we use this instead?

Comment (ssuganth) [reply gitops/openshift/apps/scss-jag/template/scss_jag.env:3]: The templates are checked-in in source code just for source controlling. Actually we use these templates to deploy manually in Openshift using OC command. The .env files with values will be fed to the template for deployment manually in openshift.

So, values from github secrets can't be used and those secrets are used only for github actions.

**Review by KFloodBCGov — APPROVED** [review]

**Review by ekwanNTT — APPROVED** [review]

**Pull request merged**